spec/inquisition/rubocop/categorizer_spec.rb

```diff
@@ -0,0 +1,40 @@
+RSpec.describe Inquisition::Rubocop::Categorizer do
+  describe '#find_category' do
```

Comment:
`describe '.find_category' do` instead of `describe '#find_category' do`
https://github.com/rubocop-hq/rspec-style-guide#example-group-naming